scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+from zipfile import ZipFile
+from collections import OrderedDict, defaultdict
+from io import StringIO
+from itertools import chain
+
+try:
+    import lxml.etree as ET
+    _have_lxml = True
+except ImportError:
+    import xml.etree.ElementTree as ET
+    _have_lxml = False
+
+import openmc.data
+import openmc.deplete
+from openmc._xml import clean_indentation
+from openmc.deplete.chain import _REACTIONS
+from openmc.deplete.nuclide import Nuclide, DecayTuple, ReactionTuple
+from openmc._utils import download
+
+from casl_chain import CASL_CHAIN
+
+URLS = [
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-neutrons.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-decay.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-nfy.zip'
+]
+
+def main():
+    if os.path.isdir('./decay') and os.path.isdir('./nfy') and os.path.isdir('./neutrons'):
+        endf_dir = '.'
+    elif 'OPENMC_ENDF_DATA' in os.environ:
+        endf_dir = os.environ['OPENMC_ENDF_DATA']
+    else:
+        for url in URLS:
+            basename = download(url)
+            with ZipFile(basename, 'r') as zf:
+                print('Extracting {}...'.format(basename))
+                zf.extractall()
+        endf_dir = '.'
+
+    decay_files = glob.glob(os.path.join(endf_dir, 'decay', '*.endf'))
+    fpy_files = glob.glob(os.path.join(endf_dir, 'nfy', '*.endf'))
+    neutron_files = glob.glob(os.path.join(endf_dir, 'neutrons', '*.endf'))
+
+    # Create a Chain
+    chain = openmc.deplete.Chain()
+
+    print('Reading ENDF nuclear data from "{}"...'.format(os.path.abspath(endf_dir)))
+
+    # Create dictionary mapping target to filename
+    print('Processing neutron sub-library files...')
+    reactions = {}
+    for f in neutron_files:
+        evaluation = openmc.data.endf.Evaluation(f)
+        name = evaluation.gnd_name
+        if name in CASL_CHAIN:
+            reactions[name] = {}
+            for mf, mt, nc, mod in evaluation.reaction_list:
+                if mf == 3:
```

Comment:
Question: Does `mf == 3` indicate that this is a neutron file? I'm basing this by my read that the `mf` value in the `reaction_list` tuple represents the file type [1], and the comment at the beginning of the loop on line `57`.

If so, it may be helpful to leave a comment explaining the significance of `3`, or making the `3` value a file-level constant.

[1] https://openmc.readthedocs.io/en/stable/pythonapi/generated/openmc.data.endf.Evaluation.html#openmc.data.endf.Evaluation